CHANGELOG.md

```diff
@@ -53,8 +54,8 @@
 - Added support for `DataFrame.tz_localize` and `Series.tz_localize`.
 - Added support for `DataFrame.tz_convert` and `Series.tz_convert`.
 - Added support for applying Snowpark Python functions (e.g., `sin`) in `Series.map`, `Series.apply`, `DataFrame.apply` and `DataFrame.applymap`.
-- Added support for `np.subtract`, `np.multiply`, `np.divide`, `np.true_divide`
-- Added support for tracking usages of `__array_ufunc__`
+- Added support for `np.subtract`, `np.multiply`, `np.divide`, and `np.true_divide`.
```

Comment:
we probably want to keep the original word, since it has already be published

Reply:
done